js/bitfinex.js

```diff
@@ -466,7 +468,7 @@ module.exports = class bitfinex extends Exchange {
         let orderType = order['type'];
         let exchange = orderType.indexOf ('exchange ') >= 0;
         if (exchange) {
-            let [ prefix, orderType ] = order['type'].split (' ');
+            [ , orderType ] = order['type'].split (' ');
```

Comment:
This one will fail in Python, unfortunately... We can't rely on the JS-undefined-behaviour.